src/Sentry/Laravel/BaseServiceProvider.php

```diff
@@ -22,7 +22,7 @@ protected function hasDsnSet(): bool
     {
         $config = $this->getUserConfig();
 
-        return !empty($config['dsn']);
+        return !empty($config['dsn']) || ($config['spotlight'] ?? false) === true;
     }
```

Comment:
What about we leave `hasDsnSet` as is and add `hasSpotlightEnabled` and centralize the checks in one additional method? Bit overkill, but cleaner. Up to you! 🙂